lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1468,13 +1469,26 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
       CapabilityIdentifier.jamesCalendarEvent.isSupported(session, accountId);
   }
 
+  @visibleForTesting
+  parseCalendarEventAction({
+    required AccountId accountId,
+    required Set<Id> blobIds,
+  }) => _parseCalendarEventAction(accountId: accountId, blobIds: blobIds);
+
   void _parseCalendarEventAction({
     required AccountId accountId,
     required Set<Id> blobIds,
-    required String emailContents
   }) {
     log("SingleEmailController::_parseCalendarEventAction:blobIds: $blobIds");
-    consumeState(_parseCalendarEventInteractor!.execute(accountId, blobIds, emailContents));
+    consumeState(_parseCalendarEventInteractor!.execute(
+      accountId,
+      blobIds,
+      TransformConfiguration.fromTextTransformers(const [
+        SanitizeAutolinkHtmlTransformers(),
+        StandardizeHtmlSanitizingTransformers(),
```

Comment:
Maybe move up this transformer

Reply:
Done

Reply:
It breaks all microsoft content.

Reply:
![Screenshot 2024-12-18 at 10 38 49](https://github.com/user-attachments/assets/4afad1e2-26ca-4df8-87ed-25ab6286c193)

Fixed